doc/source/whatsnew/v0.21.1.txt

```diff
@@ -30,7 +30,7 @@ Other Enhancements
 Deprecations
 ~~~~~~~~~~~~
 
--
+- Warnings for deprecated initialization style of ``Categorical`` (``Categorical(codes, categories)``) have been removed.
```

Comment:
Can you move to the "Removal of prior version deprecations/changes" section (in 0.22.0)